e2e/genesis_test.go

```diff
@@ -96,21 +97,18 @@ func TestCustomBlockGasLimitPropagation(t *testing.T) {
 			t.Fatalf("failed to sign txn: %v", err)
 		}
 
-		_, err = srv.TxnPoolOperator().AddTxn(context.Background(), &txpoolOp.AddTxnReq{
-			Raw: &any.Any{
-				Value: tx.MarshalRLP(),
-			},
-			From: types.ZeroAddress.String(),
-		})
-		if err != nil {
-			t.Fatalf("failed to add txn: %v", err)
-		}
+		_, err = srv.JSONRPC().Eth().SendRawTransaction(signedTx.MarshalRLP())
```

Comment:
Why the switch to JSON-RPC instead of using GRPC?

Reply:
I reverted it to gRPC calls in a couple of places. 

https://github.com/0xPolygon/polygon-sdk/pull/287/commits/e18a2cbd44e9fc2dd33872da1bbe3e921261e56a